Clover/app/src/main/java/org/floens/chan/core/presenter/ThreadPresenter.java

```diff
@@ -698,6 +707,9 @@ private void showPostInfo(Post post) {
         }
 
         // TODO(multi-site) get this from the timestamp
+        Date date = new Date () ;
+        date.setTime((long)post.time*1000);
```

Comment:
does this time when outputted respect the user's current timezone, or is it always UTC?

Reply:
It respects the user's current timezone.